master/addbootemtrigger.sql

```diff
@@ -0,0 +1,8 @@
+-- helper function to create bootstream jsonb object for handling permission scope change
+CREATE OR REPLACE FUNCTION permission_change_notify() RETURNS integer AS $$
+DECLARE
+BEGIN
+    PERFORM pg_notify('permission_change_chan');
+    return 0;
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
github is indicating that you don't have a terminating newline at the end of your file.

Can you configure your editor to always put a terminating newline at the end of the file?  That's an old unix standard behavior, and if everybody follows it then it creates less code churn when multiple people edit the same file.